src/jaegertracing/DynamicLoad.cpp

```diff
@@ -0,0 +1,26 @@
+#include <cstring>
+#include <system_error>
+
+#include <opentracing/dynamic_load.h>
+
+#include "TracerFactory.h"
+#include "jaegertracing/Tracer.h"
+
+int OpenTracingMakeTracerFactory(const char* opentracingVersion,
+                                 const void** errorCategory,
+                                 void** tracerFactory)
+{
+    if (std::strcmp(opentracingVersion, OPENTRACING_VERSION) != 0) {
```

Comment:
I'm going to assume this means the OpenTracing C++ API version. I see you use the version string to generate it. Are you sure we want to use minor and patch versions, or just major version?

Reply:
That might make sense.... though it's pretty easy to break the ABI (for example, if you were to change the order of function in the source your vtable entrys could change). It's probably safest (and also easy) to start by requiring a full match.